Cross-compilation broke somewhere between 0.9 and 0.10, we could
recommend
  http://www.sourceware.org/autobook/autobook/autobook_255.html
to upstream so they fix 'zip.h'.

Provided quick patch meanwhile.

--- a/lib/zip.h	2012-03-15 10:28:05.000000000 +0100
+++ b/lib/zip.h	2012-12-16 19:29:03.161742479 +0100
@@ -38,7 +38,7 @@
 
 #ifndef ZIP_EXTERN
 #ifdef _WIN32
-#define ZIP_EXTERN __declspec(dllimport)
+#define ZIP_EXTERN
 #else
 #define ZIP_EXTERN
 #endif
